packages/logger/tests/unit/Logger.test.ts

```diff
@@ -280,6 +280,9 @@ describe('Class: Logger', () => {
           });
           const consoleSpy = jest.spyOn(logger['console'], methodOfLogger).mockImplementation();
 
+          interface ArbitraryObject { value: 'CUSTOM' | 'USER_DEFINED' }
+          const arbitraryObject: ArbitraryObject = { value: 'CUSTOM' };
```

Comment:
Improvement: can use test cases where the value type is not a string but a nested object with different custom types? Example: Lambda events types from the `aws-lambda` library?

Reply:
Yes, we can - this is what I implemented in the E2E test. Do we also want this in the unit tests?
BTW: the type of `value` here is not `string`, but a literal type - which was not possible to use prior to this PR.

Reply:
> Yes, we can - this is what I implemented in the E2E test. Do we also want this in the unit tests?

Yes, thanks.

> BTW: the type of value here is not string, but a literal type - which was not possible to use prior to this PR.

This is what I am referring to: 
const arbitraryObject: ArbitraryObject = { value: 'CUSTOM' };

Reply:
Please take a look on the interface definition. The type of the `value` property in the object `arbitraryObject` is not `string`, it's a literal type `'CUSTOM' | 'USER_DEFINED'`. You cannot assign `value: 'SOME_OTHER'` here. And as such, this literal type cannot be expressed by anything else but `unknown` unless we exactly know it.